Src/RE2.Managed/NativeMethods.cs

```diff
@@ -34,7 +31,7 @@ static NativeMethods()
 
                 if (File.Exists(filePath))
                 {
-                    LoadLibrary(filePath);
+                    if (isLinux) { LoadLibrary(filePath); } else { LinuxMethods.dlopen(filePath, LinuxMethods.RTLD_NOW); }
                 }
             }
             else if (platform == "win")
```

Comment:
>(platform == "win") [](http://example.com/codeflow?start=20&length=19)

nit: when `platform == "win"`. `isLinux` should always be false. not needed to check `isLinux` in below if condition?